onebusaway-android/src/main/res/layout/arrivals_list_header.xml

```diff
@@ -215,6 +215,25 @@
                         android:padding="5dp"/>
             </LinearLayout>
 
+            <ImageButton
+                    android:id="@+id/stop_discussion"
+                    android:src="@drawable/es_ic_comment"
```

Comment:
I can't comment directly on the binary file, so I'll comment here instead.  Can we please change this to the material design icon for "Comment" - https://material.io/icons/#ic_comment?